lib/l10nUtil.js

```diff
@@ -87,7 +87,8 @@ module.exports.rebaseBraveStringFilesOnChromiumL10nFiles = (path) =>
         .replace('<include name="IDR_MD_HISTORY_SIDE_BAR_HTML"', '<include name="IDR_MD_HISTORY_SIDE_BAR_HTML" flattenhtml="true"')
         .replace(pageVisibility, bravePageVisibility + pageVisibility)
         .replace(/settings_chromium_strings.grdp/g, 'settings_brave_strings.grdp')
-        .replace(/Automatically sends usage statistics and crash reports to Brave/g, 'Automatically sends crash reports to Brave')
+        .replace(/Automatically send usage statistics and crash reports to Google/g, 'Automatically send crash reports to Google')
+        .replace(/Automatically sends usage statistics and crash reports to Google/g, 'Automatically sends crash reports to Google')
```

Comment:
@bbondy Two messages because the label and description differ very slightly in their wording...